whereami/whereami_payload.py

```diff
@@ -88,8 +88,15 @@ def call_http_backend(backend_service):
         def call_grpc_backend(backend_service):
 
             try:
-                channel = grpc.insecure_channel(backend_service +
-                                                ':9090')
+                # check for using `443` port to use `secure_channel`
+                # assumes port number is appended to backend_service name
+                if backend_service.split(':')[1] == '443':
```

Comment:
`443` is hard coded, could it be `8443` (less privilege) or another port too?

Reply:
i'll add `8443` and will handle the prometheus port stuff